### PR TITLE
Add links to the aspects and followed tags pages on mobile

### PR DESCRIPTION
**Pull request by Flaburgan**
Close https://github.com/diaspora/diaspora/issues/6385

### CODE REVIEW

File: features/step_definitions/user_steps.rb [hunk 1]
```diff
@@ -107,6 +107,10 @@
   user.profile.save!
 end
 
+Given /^I have a limited post with text "([^\"]*)" in the aspect "([^"]*)"$/ do |text, aspect_name|
+  post = @me.post :status_message, text: text, to: @me.aspects.where(name: aspect_name).first.id
```
Comment (diaspora-code-review) [inline features/step_definitions/user_steps.rb]: Useless assignment to variable - `post`.

Comment (Flaburgan) [inline features/step_definitions/user_steps.rb]: Well yeah, this is weird, I forgot to `post.save` here but the test is green anyway oO, [this line](https://github.com/diaspora/diaspora/pull/7265/files#diff-b0f17d137a09c6491e446c550000d479R66) should fail...

Comment (SuperTux88) [inline features/step_definitions/user_steps.rb]: I think `.post` already saves the post.

Comment (Flaburgan) [inline features/step_definitions/user_steps.rb]: Confirmed. I corrected, ready to review.

File: features/step_definitions/user_steps.rb [hunk 2]
```diff
@@ -107,6 +107,10 @@
   user.profile.save!
 end
 
+Given /^I have a limited post with text "([^\"]*)" in the aspect "([^"]*)"$/ do |text, aspect_name|
+  @me.post :status_message, text: text, to: @me.aspects.where(name: aspect_name).first.id
+end
+
```
Comment (svbergerem) [inline features/step_definitions/user_steps.rb]: Please move this to `features/step_definitions/posts_steps.rb`.

**Review by svbergerem — CHANGES_REQUESTED** [review]

**Comment by Flaburgan** [comment]
@svbergerem done.

**Review by svbergerem — APPROVED** [review]

**Comment by svbergerem** [comment]
Thanks!

**Comment by Flaburgan** [comment]
Awesome! Thank you ;)

**Pull request closed without merging**